src/models.py

```diff
@@ -54,10 +54,12 @@ class Course(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String(255))
     title = db.Column(db.String(255))
+    securityCode = db.Column(db.String(6))
```

Comment:
Yea, later on, for each course, there needs to be a random 6 dig code that will be distributed. I'm assuming we just email the code to the prof and he/she distributes it to the required students.